index.php

```diff
@@ -10,35 +32,54 @@
 
 
 $PAGE->set_context(context_system::instance());
-$PAGE->set_url('/admin/tool/api_test/index.php');
-$PAGE->set_title(get_string('pluginname', 'tool_api_test'));
+$PAGE->set_url('/admin/tool/wsformat/index.php');
+$PAGE->set_title(get_string('pluginname', 'tool_wsformat'));
 $PAGE->set_heading($SITE->fullname);
 
 echo $OUTPUT->header();
-// echo $OUTPUT->heading(get_string('pluginname', 'tool_api_test'));
-// echo get_string('plugindescription', 'tool_api_test');
-echo $OUTPUT->box_start();
 
-// Prepare the data (context) for the template
-$template_data = new stdClass(); // Create new object
+$output = $PAGE->get_renderer('tool_wsformat');
+
+$plugindescriptiontemplate = new \tool_wsformat\output\plugin_description();
+echo $output->render($plugindescriptiontemplate);
+
+use tool_wsformat\form\autocomplete_form;
```

Comment:
This use shouldnt be required in global space pages, it should already be autoloaded.